pkg/local_object_storage/pilorama/interface.go

```diff
@@ -34,6 +34,8 @@ type Forest interface {
 	// TreeGetOpLog returns first log operation stored at or above the height.
 	// In case no such operation is found, empty Move and nil error should be returned.
 	TreeGetOpLog(cid cidSDK.ID, treeID string, height uint64) (Move, error)
+	// TreeDrop drops a tree from the database.
```

Comment:
I think we should doc `ErrTreeNotFound` return case.

Also IMO `database` does not fit well under the description of the interface, because rather tends to a specific implementation. But it's not critical.

Reply:
It is tricky, because storage engine can't always return this error (write errors are more important). But, still documenting errors is a good thing, fixed.